src/lib/learn-client/types.ts

```diff
@@ -10,6 +10,11 @@ export type uuid = string
 export type slug = string
 export type identifier = uuid | slug
 
+export interface TutorialIdCollectionId {
```

Comment:
Used this type in a few places already, and I think it'll come up even more through the rest of user progress work, so felt like a decent time to elevate it to this file.